packages/ag-charts-website/src/content/docs/lazy-test/_examples/lazy/data.ts

```diff
@@ -0,0 +1,48 @@
+const dataStart = new Date('2024-01-01 00:00:00').getTime();
+const dataEnd = new Date('2024-07-01 00:00:00').getTime();
+
+const minute = 1000 * 60;
+const hour = minute * 60;
+const day = hour * 24;
+const week = day * 7;
+
+type Datum = { time: number; value: number };
+
+// Simple seeded randomisation for consistent data - https://stackoverflow.com/a/19303725
+let seed = 1234;
+function random() {
+    const x = Math.sin(seed++) * 10000;
+    return x - Math.floor(x);
+}
```

Comment:
Let's come up with our own algorithm (we have other examples which already do this too?)